tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -915,34 +917,20 @@ impl FmtSpanConfig {
     }
     pub(super) fn trace_new(&self) -> bool {
         match self.kind {
-            FmtSpan::NONE => false,
-            FmtSpan::ACTIVE => false,
-            FmtSpan::CLOSE => false,
             FmtSpan::FULL => true,
+            _ => false,
         }
     }
-    pub(super) fn trace_enter(&self) -> bool {
+    pub(super) fn trace_active(&self) -> bool {
```

Comment:
I _thought_ rustfmt would complain about missing whitespace between functions, but I guess not?

Reply:
no, it doesn’t — I tend to ensure blank lines between top-level items, but this `impl` block is light enough in appearance and contents